playbooks/tasks/xxx-move-to-portal-role.yml

```diff
@@ -87,8 +87,11 @@
   when: create_env_file_result.changed or sia_container_info_result.container == None or not sia_container_info_result.container.State.Running
 
 # xxx use community.docker.docker_prune (requires community.docker upgrade) with long timeout
+# Prune docker
 - name: Prune docker
-  command: docker system prune --force
+  ansible.builtin.command: docker system prune --force
```

Comment:
shouldn't this be before the sia docker container start?

Reply:
Added docker prune also before starting services.